ls1intum/ArTEMiS
# Translate Apollon Diagrams

### Checklist

### Motivation and Context

Translate all Apollon Diagrams pages into German so that we can offer users to switch the language.

### Description

Add German translations to Apollon Diagrams pages and listen to language changes to update Apollon locale.

### Steps for Testing


1. Log in to ArTEMiS
2. Change the Language to German in the Profile
3. Create new Drag and Drop Quiz Exercise and check for missing or wrong translations
